### DISCUSSION TITLE
Http URL Authentication

### DISCUSSION THREAD

**Discussion by jmamuedo** [fastapi/fastapi#8485] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
From fastapi import FastAPI
```

### Description

@FastAPI Hello, Why I can't get response by url when I use 'Authorize' on my endpoints? Is this only by gui way?

### Operating System

Windows

### FastAPI Version

Last

### Python Version

3.7

**Comment by nilTheDev** [comment]
Can't understand your question. There is no context. Please provide more context. Otherwise, nobody would be able to help you.

**Comment by piehld** [comment]
This test code provides some examples of authentication via code (as opposed to GUI): https://github.com/tiangolo/fastapi/blob/master/tests/test_security_http_basic_realm.py
